packages/@webex/plugin-meetings/test/unit/spec/multistream/mediaRequestManager.ts

```diff
@@ -684,6 +687,30 @@ describe('MediaRequestManager', () => {
     assert.calledOnce(sendMediaRequestsCallback);
   });
 
+  it('should not degrade max-fs if receive slot sources are not live', () => {
```

Comment:
it would be good to add a test that checks that source update triggers new degradation evaluation and new media request

Reply:
Ok, will do it in a later PR, since we want to get this in today.